src/style.css

```diff
@@ -0,0 +1,101 @@
+
+@import url("https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,400;0,500;0,700;1,300;1,400&display=swap");
+
+* {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  height: 406px;
+  background: #fff;
+  border: 1px solid #fff;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+  align-self: center;
+  margin-bottom: 40px;
+}
+
+.card__textmodel {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  font-family: "Roboto", sans-serif;
+  margin-bottom: 4px;
+  justify-content: center;
+}
+
+.card__productcode {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  margin-bottom: 16px;
+}
+
+.card__feedback {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 24px;
+  align-items: center;
+}
+
+.card__feedbackamount {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 16px;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 16px;
+}
+
+.card__textprice {
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 400;
+}
+
+.card__numberprice {
+  font-size: 16px;
+  line-height: 18px;
+  font-weight: 700;
+}
+
+.card__buybuttom {
+  text-transform: uppercase;
+  text-decoration: none;
+  color: #fff;
+  background-color: #00acdc;
+  padding: 12px 56px 12px 55px;
+  border-radius: 5px;
+  font-size: 14px;
+  font-weight: 700;
+  box-sizing: border-box;
+}
+
+.card__buybuttom:hover {
+  color: #00acdc;
+  background-color: #fff;
+  border: 1px solid #00acdc;
+  transition: 0.3s;
+}
+
+.star--staractive {
+  background-image: url(/images/star-active.svg );
+}
+
+.star {
+  background-image: url(/images/star.svg);
+ 
+}
```

Comment:
move stars block to a separate file and rewrite styles according to changes in HTML (check corresponding comment)